File: src/test/java/boat/utilities/ProcessUtilTest.java
package boat.utilities;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertNotNull;

class ProcessUtilTest {


    @Test
    void isRcloneInstalled() {
        // Given
        // When
        boolean rcloneInstalled = ProcessUtil.isRcloneInstalled();
        // Then
        assertNotNull(rcloneInstalled);
    }
}

File: src/test/java/boat/utilities/ProcessUtilTest.kt
package boat.utilities

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Disabled
import org.junit.jupiter.api.Test

internal class ProcessUtilTest {
    @Disabled
    @Test
    fun isRcloneInstalled() {
        // Given
        // When
        val rcloneInstalled = ProcessUtil.isRcloneInstalled
        // Then
        Assertions.assertTrue(rcloneInstalled)
    }
}